c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh

```diff
@@ -0,0 +1,180 @@
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+set -ex
+
+# Install dependencies
+dnf -y --disableplugin=subscription-manager install \
+    http://mirror.centos.org/centos/8.2.2004/BaseOS/ppc64le/os/Packages/centos-gpg-keys-8.2-2.2004.0.2.el8.noarch.rpm \
+    http://mirror.centos.org/centos/8.2.2004/BaseOS/ppc64le/os/Packages/centos-repos-8.2-2.2004.0.2.el8.ppc64le.rpm \
+    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+yum install -y git vim make cmake3 python2 cups-devel pkgconfig nss-devel openssl-devel glib2-devel pango-devel dbus-devel atk-devel at-spi2-atk-devel gtk3-devel krb5-devel pulseaudio-libs-devel libXScrnSaver-devel epel-release subversion curl alsa-lib-devel pciutils-devel mesa-libGLw bison patch bzip2 libuuid-devel mesa-libgbm libsecret-devel python38 python2-psutil python38-psutil java-1.*.0-openjdk-devel libXtst-devel libatomic gcc-c++ expat-devel zlib-devel perl-ExtUtils-MakeMaker wget diffutils libdrm-devel
+
+dnf install -y http://mirror.centos.org/centos/8.2.2004/PowerTools/ppc64le/os/Packages/gperf-3.1-5.el8.ppc64le.rpm
+dnf install -y http://mirror.centos.org/centos/8.2.2004/PowerTools/ppc64le/os/Packages/mesa-libgbm-devel-19.3.4-2.el8.ppc64le.rpm
+dnf install -y http://mirror.centos.org/centos/8.2.2004/PowerTools/ppc64le/os/Packages/re2c-0.14.3-2.el8.ppc64le.rpm
+dnf install -y http://mirror.centos.org/centos/8.2.2004/PowerTools/ppc64le/os/Packages/ninja-build-1.8.2-1.el8.ppc64le.rpm
+
+alternatives --set python /usr/bin/python2
+
+# Install nodejs
+curl https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+nvm install v12.13.1
+ln -s $HOME/.nvm/versions/node/v12.13.1/bin/node /usr/bin/node
+
+cd $HOME
+mkdir chromium_build
+export CHROMIUM_DIR=$HOME/chromium_build
+cd $CHROMIUM_DIR
+
+git clone https://gn.googlesource.com/gn
+export CC=gcc
+export CXX=g++
+export AR=ar
+cd gn/
+git checkout 5ed3c9cc67b090d5e311e4bd2aba072173e82db9
+python build/gen.py --no-static-libstdc++
+ninja -C out
+
+unset CXX
+unset CC
+unset AR 
+
+cd $CHROMIUM_DIR
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+
+export DEPOT_TOOLS_UPDATE=0
+export PATH=${CHROMIUM_DIR}/gn/out:$PATH:${CHROMIUM_DIR}/depot_tools
+export VPYTHON_BYPASS="manually managed python not supported by chrome operations"
+export GYP_DEFINES="disable_nacl=1"
+
+# check out LLVM and Clang
+export CLANG_REVISION=375507
```

Comment:
how do we map the chromium version to the exact clang version, if there is some commend for doing so, maybe we worth mentioning in a comment for future reference.

Reply:
Actually, there is a way to map the chromium version to clang version as used [here](https://github.com/leo-lb/electron/blob/7-2-x/ppc64le_build.bash#L85). But since I was referring to [this](https://github.com/ppc64le/build-scripts/blob/master/c/chromium/Chromium_84.0.4119.0_RHEL_7.6.sh#L116) for building chromium, I just used the same version being used there. I'll try updating that to use the command to fetch the version.